### PR TITLE
Moved Dockerfiles from oms-docker/stabilize-derk into this repo.

### PR DESCRIPTION
**Pull request by i3anaan**
https://github.com/AEGEE/oms-docker/pull/22

### CODE REVIEW

File: docker/Dockerfile.dev [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        && mkdir -p /usr/app/frontend \
+        && mkdir -p /usr/app/media \
+        && mkdir -p /usr/scripts \
+        && mkdir -p /usr/shared
+
+WORKDIR /usr/app
+
+ADD ./configFile.json /usr/scripts/configFile.json
+
+CMD cp /usr/scripts/configFile.json /usr/app/oms-events/lib/config/configFile.json && \
```
Comment (i3anaan) [inline docker/Dockerfile.dev]: Updated!

File: docker/configFile.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
+{
```
Comment (i3anaan) [inline docker/configFile.json]: The problem is that this is not available at build time, only the files inside the docker folder are included in the build context. (As far as I understand it anyway).

Maybe @linuxbandit has a better solution for this?

Comment (serge1peshcoff) [inline docker/configFile.json]: I suggest leaving it as it is for now and make an issue on JIRA about that to resolve it later.

Comment (i3anaan) [inline docker/configFile.json]: https://oms-project.atlassian.net/projects/EVENTS/issues/EVENTS-71

**Pull request merged**